INSTALL.rst

```diff
@@ -34,7 +113,8 @@ We build assets using `npm`. Make sure you have installed it system wide.
 
 .. code-block:: console
 
-    (inspirehep)$ npm update && npm install -g node-sass clean-css requirejs uglify-js
+    (inspirehep)$ sudo npm update
+    (inspirehep)$ sudo npm install -g node-sass clean-css requirejs uglify-js
```

Comment:
FWIW wouldn't it be easier to lead `npm` installation related discussions in upstream Invenio, and then simply tell the INSPIRE developers (or CDS, or Zenodo...) in the overlay instructions to run the appropriate `provision-web.sh --with-foo --with-bar`, or `provision-elasticsearch.sh` etc scripts, in order to have all the pre-requisites installed? Might be easier to maintain instructions up-to-date in this way.